src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -1,244 +1,417 @@
+GitHub issue. However, where a **Primary** is responsible for maintaining the 
+response time goal, other developers so enlisted are free to either delay or even
+decline to respond (but nonetheless inform the **Primary** of this need) if their
+schedule does not permit timely response. Such a situation could mean that the
+only remaining course of action for the **Primary** to *resolve* the issue is to
+file a product development issue as discussed at the end of a preceding section.
+
+If after investigation and diagnosis the work required to resolve a customer
+response incident remains highly uncertain or is not believed to be a
+*low-hanging-fruit* type task, the **Primary** should search the issue system to
+see if this is a known issue and, if so, add additional information to that known
+issue about this new **Customer Response** incident (and perhaps remove the
+*reviewed* tag from the issue to cause the issue to be re-reviewed at the next
+VisIt_ project meeting) or submit a *new* issue to the product development issue
+tracker. Such action then *resolves* the original **Customer Response** issue.
+
+Special Considerations for Classified Computing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Occasionally, incidents arise that may be handled only in the Secure Computing
+Facility (SCF). This is not too common but does happen and it presents problems
+for a geographically distributed team. In many ways, an SCF-only incident is just
+a different form of *escalation*.
 
 On the one hand, customers on SCF are accustomed to longer response times.
 On the other hand, often work on the SCF is a high priority and requires
 rapid response from a developer that is on site with access to SCF.
 
-In many ways, an SCF-only incident is just a different form of *escalation*.
-
 Our current plan is to handle this on a case-by-case basis. If neither the
-primary nor backup are able to handle a customer response incident requiring
-the SCF, the primary should
+**Primary** nor **Backup** are able to handle a customer response incident
+requiring the SCF, the **Primary** should
 
   * First determine the customer's required response time. It may be hours
     or it may be days. If it is days. Its conceivable the issue could be
-    handled in the following week by a new primary/backup pair.
-  * If customer indicates immediate response required, primary should inquire
-    the whole team to arrange another developer who can handle it.
-
-Relation to Programmatic Work
+    handled in the following week by a new **Primary/Backup** pair.
+  * If customer indicates immediate response is required, the **Primary**
+    should inquire the whole team to arrange another developer who can
+    handle it.
+
+Housekeeping and Low-hanging Fruit Type Issues
+----------------------------------------------
+
+Part of the reason for developing this process is the acknowledgment of the
+existence of a different category of work,
```

Comment:
I think this sentence would read better:
... this process is to acknowledge the existence of ...